lib/atom-cfn-lint.js

```diff
@@ -45,6 +45,11 @@ export default {
   // activate linter
   activate() {
     const helpers = require('atom-linter');
+
+    require('atom-package-deps').install('atom-cfn-lint')
```

Comment:
Note that this will add ~50ms to the "activation time" as reported to users in Timecop. You can schedule this work in an idle callback for Atom to take care of when it isn't doing anything else.

An example implementation of this can be [found here](https://github.com/AtomLinter/linter-eslint/blob/62b44afa4f5d876b6232b3dac04aba1081d8ff2a/src/main.js#L34-L59), note that the function names include the package name so you can easily identify them when viewing performance traces. Make sure you track the idle callbacks [somehow](https://github.com/AtomLinter/linter-eslint/blob/62b44afa4f5d876b6232b3dac04aba1081d8ff2a/src/main.js#L7-L8), and [cancel any pending](https://github.com/AtomLinter/linter-eslint/blob/62b44afa4f5d876b6232b3dac04aba1081d8ff2a/src/main.js#L197-L198) ones when your package is `deactivate`ed 😉.